Headers/AppKit/NSText.h

```diff
@@ -69,13 +69,22 @@
 @class NSColor;
 @class NSFont;
 
-typedef enum _NSTextAlignment {
+typedef NSInteger NSTextAlignment;
+enum {
   NSLeftTextAlignment = 0,
   NSRightTextAlignment,
   NSCenterTextAlignment,
   NSJustifiedTextAlignment,
   NSNaturalTextAlignment
-} NSTextAlignment;
+};
+
+enum {
```

Comment:
Could you please make this conditional on OS_API_VERSION(MAC_OS_X_VERSION_10_11? This makes it clearer when the renaming happened.